modules/programs/notmuch-virtual-mailbox.nix

```diff
@@ -0,0 +1,33 @@
+{ config, lib, ... }:
```

Comment:
This file doesn't seem to be referenced by any email modules

Reply:
It is here: https://github.com/nix-community/home-manager/pull/3143/files#diff-534d37110e14d17f12ecce48a24f06987d11dd77b4b02218702186d29d708173R31